coalib/output/printers/LogPrinterMixin.py

```diff
@@ -0,0 +1,45 @@
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
```

Comment:
just to be clear, I meant removing LogPrinterMixin.py, but you (or I? :)) are right, the LOG_LEVEL module can be removed as well.